mysql-test/suite/rocksdb.sys_vars/t/rocksdb_delayed_write_rate_basic.test

```diff
@@ -0,0 +1,22 @@
+--source include/have_rocksdb.inc
```

Comment:
All of the MyRocks sysvars tests need this actually, not just these new tests. Rather than fixing this here, I created a new issue (https://jira.percona.com/browse/MYR-146) and will send in a pull request that fixes all of these tests.